autoload/importjs.vim

```diff
@@ -251,17 +258,18 @@ function! importjs#Init()
     let s:job_executable='importjs'
   endif
 
-  " neovim
   if exists("*jobstart")
+    " neovim
     let s:job = jobstart([s:job_executable, 'start', '--parent-pid', getpid()], s:callbacks)
-  endif
-
-  " vim
-  if exists("*job_start")
+  elseif exists("*job_start")
+    " vim
     let s:job=job_start([s:job_executable, 'start', '--parent-pid', getpid()], {
           \'exit_cb': 'importjs#JobExit',
           \})
 
     let g:ImportJSChannel=job_getchannel(s:job)
+    " ignore first line of output, which is something like
+    " > ImportJS (v2.10.1) DAEMON active.
+    call ch_readraw(g:ImportJSChannel, { "timeout": 2000 })
```

Comment:
The reason we don't need this in the neovim path is that it ignores anything that doesn't start with `"{"`.